packages/orbit-components/src/ListChoice/index.js

```diff
@@ -81,8 +81,17 @@ StyledListChoiceContent.defaultProps = {
   theme: defaultTheme,
 };
 
-const ListChoice = (props: Props) => {
-  const { dataTest, icon, title, description, selectable, onClick, selected, disabled } = props;
+const ListChoice = ({
+  dataTest,
+  icon,
+  title,
+  description,
+  selectable,
+  onClick,
+  selected,
+  disabled,
+  tabIndex = 0,
```

Comment:
Considering that this seems to be a separate feature, add it to a separate commit so that it's visible in the changelog.